src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -237,6 +237,31 @@ def cancelWorkflow(self, wf):
         elements = [x['id'] for x in data.get('rows', []) if x['key'][1] not in nonCancelableElements]
         return self.updateElements(*elements, Status='CancelRequested')
 
+    def updateSiteLists(self, wf, siteWhiteList, siteBlackList):
+        """
+        Update site lists of a workflow
```

Comment:
Can you please improve this docstring? Please refer to https://github.com/dmwm/WMCore/blob/master/CONTRIBUTING.rst#project-docstrings-best-practices

In addition, I think we should make both sitewhitelist and siteblacklist optional, as a workflow could be updating only one of those lists.

Reply:
I'm not sure what to improve as docstring complaint with contributing requirements, i.e. it provides description of input parameters and what is returned by the function. If further improvements in docstring itself is required please be specific.

Meanwhile, I converted site/black lists to be optional parameters.

Reply:
Short description of the method can likely be better, e.g. "Update site list parameters in elements matching a given workflow and a list of element statuses.
In addition, parameters are supposed to mention the data type, as written in the document I linked above.